setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = sunpy
-package_name = sunpy # astropy compat
```

Comment:
This comment is here to indicate we should remove it at somepoint as it is not specced by setuptools, but used by some astropy tooling like `sphinx-astropy`

Reply:
But we can just leave it there forever.

Reply:
WE WILL DELETE IT WHEN THE TIME COMES